generated/nidaqmx/_grpc_interpreter.py

```diff
@@ -1,10 +1,12 @@
 # Do not edit this file; it was automatically generated.
 
 import grpc
+import nidaqmx
 import typing
 import warnings
 
 from . import errors as errors
+from . import error_codes
 from nidaqmx._base_interpreter import BaseInterpreter
```

Comment:
These new imports are not used, so delete them.

Reply:
deleted the unused imports